### PR TITLE
Adding issue triage to how-to

### PR DESCRIPTION
**Pull request by CathPag**

### CODE REVIEW

**Review by jihoon-seo — COMMENTED** [review]
@CathPag Thank you for your devotion!

I left some comment.
PTAL! 😊

**Comment by CathPag** [comment]
Thanks for your great suggestions @jihoon-seo :) 
I uploaded the images here but they don't appear. Can you help?

**Comment by CathPag** [comment]
Never mind. I saw your comment in the other issue that it was the file name :)  If this looks good to go, can you approve please?

File: content/en/contribute/_index.md [hunk 1]
```diff
@@ -11,15 +11,33 @@ The Cloud Native Glossary content is stored in [this GitHub repo](https://github
...
+Consider joining our monthly Glossary Working Group meetings if you want to contribute regularly. You can find meeting details in the [CNCF calendar](https://www.cncf.io/calendar/). You can also connect with the maintainers and fellow contributors in our [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) channel on the CNCF Slack — we'd love to meet you! 
+
+## Work on an existing issue
+Go to the [Glossary GitHub repo issues](https://github.com/cncf/glossary/issues). There you'll see a list of all issues. You can filter by label (e.g. English language, help needed, first good issue). Note that you'll need a GitHub account to do any of this.
+
+![Issue and labels](/images/how-to/issue-and-labels.png)
+
+Make sure the term you are interested in is not already assigned to someone. Here you can see that the first three terms are available while the next term has alrady been assigned.
```
Comment (seokho-son) [inline content/en/contribute/_index.md:27]:
```suggestion
Make sure the term you are interested in is not already assigned to someone. Here you can see that the first three terms are available while the next term has already been assigned.
```

File: content/en/contribute/_index.md [hunk 2]
```diff
@@ -11,15 +11,33 @@ The Cloud Native Glossary content is stored in [this GitHub repo](https://github
...
+
+![assigning a term](/images/how-to/howto-04.png)
+
+Once you found a term you want to work on, say so in the issue. Click on it and add a comment.
+
+![Claiming an issue](/images/how-to/claiming-an-issue.png)
+
+Additionally, please also join the [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) channel on the CNCF Slack and let the maintainers know that you've raised an issue for a new term (ideally, tag _@Catherine Paganini_, _@jmo_, _@Seokho Son_, _@Jihoon Seo_, and/or _@iamnoah_ to be sure they don't miss it). Note that you can only claim one term at a time. If you want to work on multiple terms, please finish one before claiming the next one.
+
+Once they assign it to you, you can start working on it. For the next steps, please refer to the "Submitting a new term (creating a PR)" section.
```
Comment (seokho-son) [inline content/en/contribute/_index.md:37]:
```suggestion
Once they assign it to you, you can start working on it. For the next steps, please refer to the [Submitting a new term (creating a PR)](#submitting-a-new-term-creating-a-pr) section.
```
link to header

**Review by seokho-son — COMMENTED** [review]
@CathPag it looks good to me in general :)

I think we'd better to use the other issue template for proposing a new term.

The current examples (screenshots) are using `English Language Glossary Request` template which dose not includes informative contents. (https://github.com/cncf/glossary/issues/new?assignees=&labels=english&template=en.yml&title=%5BEnglish%5D%3A+GlossaryWordHere)

The latest template includes some guidelines to make applicants provides some contexts for a  suggested term. `Request to add a new term (Default:English)`

https://github.com/cncf/glossary/issues/new?assignees=&labels=lang%2Fen%2Cnew+term&template=new-term.yml&title=%5BNew+term%5D+%60GlossaryTermHere%60+for+%60En%60

Also, the template assigns predefined labels automatically. (currently, `lang/en` and `new term`)
Maybe we can update the template with more appropriate labels (such as `awaiting-triage`)

My suggestion regarding issue templates as follows.
 - removing `English Language Glossary Request` template
 - guiding contributors use `Request to add a new term (Default:English)` template
   - updating default labels for `Request to add a new term (Default:English)`

If we agree with the suggestion, I will open a PR to update issue templates.

**Comment by iamNoah1** [comment]
> @CathPag the changes sound good to me in general :)
> 
> I have an opinion regarding issue template (the template we are using in the guide)
> 
> I think we'd better to use the other issue template for proposing a new term.
> 
> The current examples (screenshots) are using `English Language Glossary Request` template which dose not includes informative contents. (https://github.com/cncf/glossary/issues/new?assignees=&labels=english&template=en.yml&title=%5BEnglish%5D%3A+GlossaryWordHere)
> 
> The latest template includes some guidelines to make applicants provides some contexts for a suggested term. `Request to add a new term (Default:English)`
> 
> https://github.com/cncf/glossary/issues/new?assignees=&labels=lang%2Fen%2Cnew+term&template=new-term.yml&title=%5BNew+term%5D+%60GlossaryTermHere%60+for+%60En%60
> 
> Also, the template assigns predefined labels automatically. (currently, `lang/en` and `new term`) Maybe we can update the template with more appropriate labels (such as `awaiting-triage`)
> 
> My suggestion regarding issue templates as follows.
> 
> * removing `English Language Glossary Request` template
> * guiding contributors use `Request to add a new term (Default:English)` template
>   
>   * updating default labels for `Request to add a new term (Default:English)`
> 
> If we agree with the suggestion, I will open a PR to update issue templates.

agreed :)

**Review by iamNoah1 — APPROVED** [review]
Awesome @CathPag :) lgtm

File: content/en/contribute/_index.md [hunk 3]
```diff
@@ -28,19 +46,18 @@ Go to the [Glossary GitHub repo](https://github.com/cncf/glossary/issues) issues
...
+You'll see a variety of templates. To propose a new term in English, select "Request to add a new term (Default:English)."
 
-![templates](/images/how-to/howto-02.png)
+![templates](/images/how-to/english-issue-template-new.png)
 
-Add the word you're suggesting and hit "submit new issue." If you're just proposing a new term, you're done! To work on it, follow the next steps.
+Add the word you're suggesting, answer the two questions below, and hit "submit new issue." If you're just proposing a new term, you're done! To work on it, follow the next steps.
 
-![new issue](/images/how-to/howto-03.png)
+![new issue](/images/how-to/submit-issue.png)
```
Comment (seokho-son) [inline content/en/contribute/_index.md:55]:
```suggestion
```
I think this screenshot can be removed :)

**Review by seokho-son — APPROVED** [review]
LGTM :)

File: content/en/contribute/_index.md (hunk 3 above)
Comment (iamNoah1) [reply content/en/contribute/_index.md]: nice catch :)

**Pull request merged**